Sandoval_David_Emanuel/005/A005.py

```diff
@@ -0,0 +1,33 @@
+# 005-
+#     Función que reciba como parametros dos números (a y b), y
+# devuelva una tupla con todos los números primos que se encuentra entre
+# a y b (incluyendo a y b si son primos).
+#     Por defecto a=0, y b=100
+
+
+def primos_en_rango(a=0, b=100):
+    primos = [2]  # lista de números primos
+    # recorro todos los números hasta b
+    for num in range(2, b + 1):
+        for primo in primos:
+            # divido cada número por cada número primo
+            if num % primo == 0:
+                # si la división da cero, no es primo
+                # de modo que corto el loop, 
+                # no necesito seguir dividiendo
+                break
+        else:
+            # Si lo anterios no sucede, el número es primo
+            # lo agrego a la lista de primos.
+            # (Este primo también va a ser utilizado para 
+            # chequear los siguientes números)
+            primos.append(num)
+
+    # Solo incluyo los primos dentro del rango indicado
+    primos_in_range = tuple([p for p in primos if (p >= a and p <= b)])
```

Comment:
Se podría en el primer for empezar a buscar números primos a partir de a, para no tener que volver recorrer los números encontrados.

Reply:
De hecho, había hecho una versión que hacía justamente eso, en el primer for, luego de saber si era primo, chequeaba si ese primos estaba dentro del rango indicado, y lo ponía en una lista aparte: primos_in_range.